backend/data/src/main/kotlin/io/tolgee/dtos/request/UserMfaRecoveryRequestDto.kt

```diff
@@ -7,6 +7,6 @@ import jakarta.validation.constraints.Size
 @JsonIgnoreProperties(ignoreUnknown = true)
 data class UserMfaRecoveryRequestDto(
   @field:NotBlank
-  @field:Size(max = 50)
+  @field:Size(max = 200)
```

Comment:
That's bad; the BCrypt algorithm has a limit before which the password gets truncated. This limit is specified to be 56 bytes, although a lot of implementations have different interpretations of the full wording used. The safe take is that any decent BCrypt implementation supports password up to 50 characters.

Any character past this is effectively ignored; assuming a string `a` of ~50 characters, `a + "b"`  would be a "valid password" in the eyes of BCrypt, the extra byte being ignored.

Reply:
Completely forgot about this.